### PR TITLE
validation of orientation

### PR DESCRIPTION
**Pull request by kd0aij**
this seems to be working correctly for both RPY and quaternion orientation specs, but displayed axes are body frame with Z down

### CODE REVIEW

**Comment by RomanBapst** [comment]
Cool, thanks, will test this today. I also added another frame which represents desired attitude. Really cool to see the error visually.

File: plot_flight_maneuver.py [hunk 1]
```diff
@@ -90,23 +174,41 @@ def animate(self,i):
         ax.set_xlabel('X')
         ax.set_ylim3d([self.y[self.INDEX[self.frame]]-1.0, 1.0+self.y[self.INDEX[self.frame]]])
         ax.set_ylabel('Y')
-        ax.set_zlim3d([-self.z[self.INDEX[self.frame]]-1.0, - self.z[self.INDEX[self.frame]]+1.0])
+        ax.set_zlim3d([self.z[self.INDEX[self.frame]]-1.0, self.z[self.INDEX[self.frame]]+1.0])
         x = []
```
Comment (RomanBapst) [inline plot_flight_maneuver.py:189]: I think you should add here: ax.invert_zaxis()
Then you just have to set the sign of the z coordinates such that the nose is up and all should be ok

**Comment by RomanBapst** [comment]
Merged

**Comment by kd0aij** [comment]
just added the invert_z_axis, seems to work fine

**Comment by RomanBapst** [comment]
Cool, can you pull master and check if it's right or if I messed up the coordinates. Your changes should all be in except maybe that I messed up the airframe coordinates. I alos added the desired attitude plot

**Comment by kd0aij** [comment]
will have to generate new log files to test; will flash latest master on my Caipi and do a tethered test... we've had over a foot of snow in the last week, and it's still snowing now :)

**Comment by RomanBapst** [comment]
ah yeah, sorry I forgot about that, PX4 master does not log quaternion setpoint yet :-) I was using my own log and they have it logged. Anyway I plan to push the VTOL controller soon anyway.

**Pull request closed without merging**